plugins/register-component/src/components/RegisterComponentForm/RegisterComponentForm.test.tsx

```diff
@@ -37,7 +37,7 @@ describe('RegisterComponentForm', () => {
     const { rendered } = setup();
     expect(
       await rendered.findByText(
-        'Enter the full path to the component.yaml file in GitHub to start tracking your component. It must be in a public repo.',
+        'Enter the full path to the component.yaml file in GitHub, GitLab, Bitbucket or Azure to start tracking your component. For private repo provide authentication information via config.',
```

Comment:
I think you could leave out the "For private..." part here. This is not actionable for the end user. Maybe it would be better if the backend threw an error with a message along the lines of "Failed to read from private repository because the Azure plugin was not configured with an API secret." That would not give unnecessary information to users that are on properly configured systems, and still gives enough information to give to the team that owns Backstage to know where to dig into addressing it.